dependencies.yaml

```diff
@@ -266,9 +279,18 @@ dependencies:
           # Not sure where these go, if anywhere:
           # - openslide
           # - xorg-libxcb
+    specific:
       - output_types: [requirements, pyproject]
-        packages:
-          - cupy-cuda11x>=12.0.0
+        matrices:
+          - matrix: {cuda: "12.*"}
+            packages:
+              - cupy-cuda12x>=12.0.0
+          - matrix: {cuda: "11.*"}
+            packages:
+              - &cupy_cu11 cupy-cuda11x>=12.0.0
+          - matrix:
+            packages:
+              - *cupy_cu11
```

Comment:
unfamiliar with the notation, curious what the `&` and `*` in the package names do here?

Reply:
These are YAML anchors. We defined an anchor with & and reused its value with *.

Reply:
oh sure, happy to explain! This is a YAML "alias", pointing to a YAML "anchor". When this document is loaded, `*cupy_cu11` will be replaced with whatever value is tagged with `&cupy_cu11`. It's a way to reduce a bit of duplication, to keep things that should be consistent consistent.

For example, this:

```yaml
limits:
  cpu: 1.0
requests:
  cpu: 1.0
```

Is identical to this:

```yaml
limits:
  cpu: &cpu_val 1.0
requests:
  cpu: *cpu_val
```

For reference, see:

* https://support.atlassian.com/bitbucket-cloud/docs/yaml-anchors/
* https://yaml.org/spec/1.2.2/#3222-anchors-and-aliases